p2p/peerManager/peerdb/counter_test.go

```diff
@@ -7,17 +7,18 @@ import (
 	"testing"
 
 	"github.com/dominant-strategies/go-quai/cmd/utils"
-	"github.com/ipfs/go-datastore"
+	datastore "github.com/ipfs/go-datastore"
 	"github.com/spf13/viper"
 	"github.com/stretchr/testify/require"
 )
 
 func TestCounter(t *testing.T) {
 	viper.GetViper().Set(utils.DataDirFlag.Name, os.TempDir())
 	dbDir := "testdb"
+	locationName := "zone-0-0"
```

Comment:
Can we use the same function to generate locationName as in other places? Rather than hardcoding.